src/DataLogger/DataLogger.h

```diff
@@ -0,0 +1,58 @@
+#ifndef _DATA_LOGGER_H_
+#define _DATA_LOGGER_H_
+
+#include "Arduino.h"
+#include "SPI.h"
+#include <SD.h>
+using namespace std;
+
+//Class to log data to a file on an SD Card in the CSV format.
+class DataLogger {
+    public:
+
+        //Constructs a DataLogger object and begins access to the SDcard.
```

Comment:
pls format comments as in other files in this project:

```
/**
 * @brief (description here)
 * ...
*/
```

This formatting offers many options for adding documentation on return types/params/etc.. and will format it all for you.  Try going to main.cpp and hovering over one of these method calls to see what I mean:

```
tft.begin();
tft.setRotation(3);
tft.fillScreen(COLOUR_BLACK);
```

These aren't from our source code, but they do use the same style of documentation